src/util/jwt.ts

```diff
@@ -0,0 +1,21 @@
+import { jwtDecode } from "jwt-decode";
+
+interface JwtPayload {
+  roles: string[];
+  exp: number; // in seconds
+}
+
+export function verifyJwt(jwt: string) {
+  let decodedToken;
+  try {
+    decodedToken = jwtDecode<JwtPayload>(jwt);
```

Comment:
Is there no way to do `GET /auth/roles/` or something similar? Would prefer to not decode tokens locally as client should assume it's a magic token (this will also make changing from jwts much easier if we ever do it)